decidim-core/config/locales/en.yml

```
@@ -725,6 +735,7 @@ en:
       others: "%{count} more"
       title: Liked by
       you: you
+    ephemeral_user: Visitant
```

I would use "Visitor" or "Guest". 
```suggestion
    ephemeral_user: Visitor
```

---

That was my Spanglish :sweat_smile: